src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Tools.Migrate
+{
+    public partial class MigrateCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false);
```

Comment:
`throwOnUnexpectedArg: false` and `AllowArgumentSeparator = true` are used when your command is passing through parameter to the underlying app (dotnet run passing through to the users app, dotnet test passing through to the runner).

I don't think we need those here.